cmd/start/cache.go

```diff
@@ -14,8 +14,7 @@
 //
 // In addition to function deleteDirectoryContentsSpareCache a function
 // is provided to clean the cache. This is done in two functions to avoid
-// making the code too complex as new cache cleaning functionality will
-// likely be added in the future, for example to implement build skipping.
+// making the code too complex.
```

Comment:
It appears unrelated to the removal of the wrappers. Is this intended?

Reply:
Yes the changes in this file are unrelated. I needed to understand the caching / cleanup code in order to know where to place the Aqua scanner binary and the modified keystone. While doing that I adjusted some code and comments based on my gained understanding.